### ISSUE TITLE
Deal with timestamps for solo clients

### ISSUE THREAD

**Issue by cwgoes** [cosmos/ibc#426]
Ref https://github.com/cosmos/cosmos-sdk/pull/6267#discussion_r429238161

**Comment by colin-axner** [comment]
should the header signature also sign over a timestamp?

**Comment by cwgoes** [comment]
> should the header signature also sign over a timestamp?

Would we use it for anything? There is no unbonding/trusting period for solo machines.

**Comment by colin-axner** [comment]
No, but we are updating the consensus state so it might be better to be more consistent with time instead of using the timestamp from the last consensus state?

**Comment by cwgoes** [comment]
> No, but we are updating the consensus state so it might be better to be more consistent with time instead of using the timestamp from the last consensus state?

Sure, that seems reasonable.

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
